src/Std/System/Uri.lean

```diff
@@ -0,0 +1,115 @@
+/-
+Copyright (c) 2019 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Lovett
+-/
+
+namespace System
+namespace Uri
+namespace UriEscape
+
+/- https://www.ietf.org/rfc/rfc3986.txt -/
+def Zero : UInt8 := '0'.toNat.toUInt8
+def Nine : UInt8 := '9'.toNat.toUInt8
+def Lettera : UInt8 := 'a'.toNat.toUInt8
+def Letterf : UInt8 := 'f'.toNat.toUInt8
+def LetterA : UInt8 := 'A'.toNat.toUInt8
+def LetterF : UInt8 := 'F'.toNat.toUInt8
```

Comment:
Should be lower-case and likely `private`, though I would probably just inline them

Reply:
@lovettchris Could you please inline these functions?

Reply:
Done.